From 33e7e89c395b4a94f53d7be2772c7051e030f224 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 19 Sep 2010 19:36:57 +0000
Subject: [PATCH 159/160] command-list.txt: list git-sh-i18n under purehelpers
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change command-list.txt to list git-sh-i18n under the purehelpers
category along with git-sh-setup.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 command-list.txt |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/command-list.txt b/command-list.txt
index 95bf18c..93e24ec 100644
--- a/command-list.txt
+++ b/command-list.txt
@@ -111,6 +111,7 @@ git-show                                mainporcelain common
 git-show-branch                         ancillaryinterrogators
 git-show-index                          plumbinginterrogators
 git-show-ref                            plumbinginterrogators
+git-sh-i18n                             purehelpers
 git-sh-setup                            purehelpers
 git-stash                               mainporcelain
 git-status                              mainporcelain common
-- 
1.7.3.272.g31195

